utils/crypto/setEnv.sh

```diff
@@ -11,3 +18,8 @@ TS_NODE=$(yarn --modules-folder $MOD_DIR bin ts-node)
 
 # execute what ts-node has printed out
 eval $($TS_NODE $DIR/setEnv.ts "$@")
+
+# reactivate -x flag for cmd printing
+if [[ $HAS_X_FLAG == true ]]; then
```

Comment:
This would fail with unbound variable error if `x` flag wasn't set, since the upstream scripts that source also have `set -u` flag set.

Either `HAS_X_FLAG` needs to be assigned to something before the first `if` statement, or this needs to be changed to

```suggestion
if [[ "${HAS_X_FLAG-}" == true ]]; then
```